src/core/startCoreFactory.js

```diff
@@ -50,13 +51,19 @@ function startCoreFactory(
       'core',
       coreCommand,
       [
-        '--publish=20002:20002',
+        `--publish=${config.get('core.rpc.port')}:${config.get('core.rpc.port')}`,
```

Comment:
We already publish RPC port by default https://github.com/dashevo/mn-bootstrap/pull/150/files#r507904035

Reply:
Got stuck here for a long time, turns out `docker-compose run` doesn't automatically publish ports. I've replaced explicit port publishing with `--service-ports`. The RPC port is definitely required for the generate and register commands, and also for the status command (as described in the PR description).